index.html

```diff
@@ -4,9 +4,75 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Microverse Portfolio Page</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="styles.css">
+    <title>Microverse Personal Portfolio Page</title>
 </head>
 <body>
-    <h1>I'am Abdul Salam</h1>
+    <header>
+        <nav id="navbar">
+
+            <a href="#" id="logo">My Logo</a>
+            <nav>
+                <div id="manubar-container">
+                    <div id="menu-bars">
+                        <div class="menu-bar"></div>
+                        <div class="menu-bar"></div>
+                        <div class="menu-bar"></div>
+                      </div>
+                </div>
+            </nav>
+        </nav>
+       
+    </header>
```

Comment:
- [ ] Nice work wrapping the "Toolbar" section in a `header` :100:  However it would be nice to remove the `<nav id="navbar">` from the header. For semantics reasons it's advisable to use `nav` tag for only navigation links.
```suggestion
    <header>
            <a href="#" id="logo">My Logo</a>
            <nav>
                <div id="manubar-container">
                    <div id="menu-bars">
                        <div class="menu-bar"></div>
                        <div class="menu-bar"></div>
                        <div class="menu-bar"></div>
                      </div>
                </div>
            </nav>
       
    </header>
```